### DISCUSSION TITLE
create AWS database error

### DISCUSSION THREAD

**Discussion by Ho-ChunHuang-NOAA** [dtcenter/METplus#1574] [Plot Generation] [answered]
Hi, 
I try to load new database on the AWS, the load script was working last week but it failed today with additional requirement "passwd?"  The new statistic did ftp to AWS server.  Is there any changes within a week since last Monday? 
ERROR MSG
::::::::::::::::::::::::::::::::::
::::::::::::::::::::::::::::::::::::
PROD_BC_PMMAX_20190809.stat                                                                          100%  580KB   7.7MB/s   00:00
PROD_AQ_20190809.stat                                                                                100%    0     0.0KB/s   00:00
PROD_BC_PMMAX_20190824.stat                                                                          100%  579KB   7.7MB/s   00:00
PROD_BC_PMMAX_20190820.stat                                                                          100%  579KB   7.7MB/s   00:00
PROD_PMMAX_20190810.stat                                                                             100%  576KB   7.7MB/s   00:00
PROD_PM_20190829.stat                                                                                100%    0     0.0KB/s   00:00
PROD_PM_20190825.stat                                                                                100%    0     0.0KB/s   00:00
PROD_BC_AQ_20190801.stat                                                                             100%    0     0.0KB/s   00:00
PROD_BC_AQMAX_20190811.stat                                                                          100%  823KB   9.7MB/s   00:00
PROD_PMAVE_20190827.stat                                                                             100%  574KB  10.4MB/s   00:00
PROD_BC_AQMAX_20190816.stat                                                                          100%  825KB  10.7MB/s   00:00
PROD_PMMAX_20190804.stat                                                                             100%  575KB  10.8MB/s   00:00
PROD_BC_AQ_20190830.stat                                                                             100%    0     0.0KB/s   00:00
PROD_PM_20190817.stat                                                                                100%    0     0.0KB/s   00:00
PROD_AQ_20190827.stat                                                                                100%    0     0.0KB/s   00:00
PROD_AQMAX_20190806.stat                                                                             100%  820KB  10.3MB/s   00:00
PROD_BC_AQ_20190809.stat                                                                             100%    0     0.0KB/s   00:00
CALLING: scp /gpfs/dell2/emc/modeling/noscrub/Ho-Chun.Huang/METviewer_AWS/XML/load_g2o_met_icmaq.xml ho-chun.huang@metviewer.nws.noaa.gov:~

You are accessing a U.S. Government information
system, which includes: 1) this computer, 2) this
computer network, 3) all computers connected to this
network, and 4) all devices and storage media attached
to this network or to a computer on this network. You
understand and consent to the following: you may
access this information system for authorized use only;
you have no reasonable expectation of privacy regarding
any communication of data transiting or stored on this
information system; at any time and for any lawful
Government purpose, the Government may monitor,
intercept, and search and seize any communication or
data transiting or stored on this information system; and
any communications or data transiting or stored on this
information system may be disclosed or used for any
lawful Government purpose.

load_g2o_met_icmaq.xml                                                                               100% 1033    40.3KB/s   00:00

You are accessing a U.S. Government information
system, which includes: 1) this computer, 2) this
computer network, 3) all computers connected to this
network, and 4) all devices and storage media attached
to this network or to a computer on this network. You
understand and consent to the following: you may
access this information system for authorized use only;
you have no reasonable expectation of privacy regarding
any communication of data transiting or stored on this
information system; at any time and for any lawful
Government purpose, the Government may monitor,
intercept, and search and seize any communication or
data transiting or stored on this information system; and
any communications or data transiting or stored on this
information system may be disclosed or used for any
lawful Government purpose.

INFO:root:METdbload Version: 1.0.0
INFO:root:--- *** --- Start METdbLoad --- *** ---
INFO:root:Begin time: 2022-04-19 14:14:41.805571
WARNING:root:!!! XML must include user and passsword tags
WARNING:root:!!! Unknown tag: met_version
INFO:root:database name is: mv_g2o_met_icmaq_aug19
INFO:root:Initial number of files: 0
WARNING:root:!!! No files to load
*** No files to load

LOAD script
<load_spec>
  <connection>
    <host>metviewer-dev-2-cluster.cluster-c0bl5kb6fffo.us-east-1.rds.amazonaws.com:3306</host>
    <database>mv_g2o_met_icmaq_aug19</database>
    <user>rds_user</user>
    <password>rds_pwd</password>
    <management_system>aurora</management_system>
  </connection>

  <met_version>V10.0.1</met_version>

  <verbose>true</verbose>
  <insert_size>1</insert_size>
  <mode_header_db_check>true</mode_header_db_check>
  <stat_header_db_check>true</stat_header_db_check>
  <drop_indexes>false</drop_indexes>
  <apply_indexes>true</apply_indexes>
  <load_stat>true</load_stat>
  <load_mode>true</load_mode>
  <load_mpr>true</load_mpr>
  <load_orank>true</load_orank>
  <force_dup_file>true</force_dup_file>
  <group>NCEP_hcHuang</group>
  <description>Loading mv_g2o_met_icmaq_aug19</description>
  <folder_tmpl>/base_dir/{type}</folder_tmpl>
    <load_val>
     <field name="type">
       <val>verif_icmaq</val>
     </field>
    </load_val>
  <load_xml>true</load_xml>
  <load_note></load_note>
</load_spec>

**Comment by Ho-ChunHuang-NOAA** [comment, marked as answer]
This discussion window changes the format of text for the xml file, please see https://www.emc.ncep.noaa.gov/mmb/hchuang/methelp/load_g2o_met_icmaq.xml  for user and passwd tag enclosure

**Reply by Ho-ChunHuang-NOAA** [reply]
To Whom It May Concern, 

Today I can create a new database on the AWS.  Please hold on your investigation till I can make sure it is normal again.

**Reply by Ho-ChunHuang-NOAA** [reply]
I am able to  upload two database today and use them both producing graphic.  Please ignore my issue.  Thank you for your attention.

**Reply by j-opatz** [reply]
Thanks @Ho-ChunHuang-NOAA for letting us know about the problem and solution to the database issue you were having. I'm going to mark this as solved and close the Discussion.
